README.md

```diff
@@ -33,6 +33,24 @@ npm install
 npm run init
 ```
 
+### Client-side Javascript (UMD modules)
+
+Running :
+
+```npm run web-build```
+
+will generate multiple files in a ```dist``` folder. They are UMD modules 
+that can be loaded in a client-side script. However, to reduce file sizes for 
+those who just need a specific syntax for the parser, there is one file per syntax.
+
+To add a syntax to the parser (ie. less) your script should start with : 
+
+```
+var gonzales = require("gonzales-core");
+
+require("gonzales-less")(gonzales);
+```
+
```

Comment:
I find this a bit impractical.
All parser methods are synchronous and code is heavy, that's why I would never recommend to run it in a browser: instead, it will be better to run a server and make requests to it.
But if for some reason you have no other choice (*and I still don't see how this can happen!*), you should at least use web worker for this task.
So... this thing should work as a worker and not as a `require` module.